ci/meta.yml

```diff
@@ -102,7 +102,7 @@
               terraform workspace show
               cp ../terraform-bootstrap/terraform.tf .
               terraform init -upgrade
-              terraform plan -out terraform.plan
+              terraform plan -out terraform.plan -replace=aws_autoscaling_schedule.test_on[0] -replace=aws_autoscaling_schedule.test_off[0]
```

Comment:
Can you explain this one please. Not sure what it is for.

Reply:
This forces re-creation of the schedules for startup and shutdown of the ecs clusters to aid with testing